modules/tinymce/src/themes/silver/main/ts/ui/context/ContextFormSlider.ts

```diff
@@ -49,6 +49,9 @@ export const renderContextFormSliderInput = (
         }
       }),
       AddEventsBehaviour.config('slider-events', [
+        AlloyEvents.runOnAttached((comp) => {
```

Comment:
I think we would want this to work like the other onSetup APIs where it has a detached function as well if you check in the silver ui/controls/Control.ts there is onControlAttached/onControlDetached one could maybe use that module for this control or replicate that behavior so that you can do:
```
onSetup: (api) => {
  const something = () => {
     console.log('hello');
  };

  editor.on('event', something);

  return () => {
    editor.off('event', something);
  };
}
```

Then we can remove the two weird events I added to check when the context toolbar is removed and migrate to this instead.

Reply:
Hmm, you right there is a pattern like that in other onSetups 🤔 what weird events do you mean?

Reply:
That was just an example of usage of the destroy/detach handler.